docs/API/AttributesUsage.md

```diff
@@ -0,0 +1,227 @@
+# Attributes usage
+
+## Overview
+
+Attributes is a feature that allows chaincode to make use of extended data in a transaction certificate. These attributes are certified by the Attributes Certificate Authority (ACA) so the chaincode can trust in the authenticity of the attributes' values.
+
+To view a complete documentation about attributes design please read ['Attributes support'](../tech/attributes.md).
+
+## Use case: Authorizable counter
+
+A common use case to make use of attributes is Attributes Based Access Control (ABAC). This is a mechanism to grant specific permissions to a chaincode invoker based on its attribute values.
```

Comment:
A common use case for the Attributes feature is Attributes Based Access Control (ABAC).